charts/cortx/templates/_config.yaml.tpl

```diff
@@ -13,9 +13,16 @@
 {{- range $i := until (int .Values.cortxdata.replicas) -}}
 {{- $dataHostnames = append $dataHostnames (printf "%s-%d.%s" (include "cortx.data.fullname" $) $i (include "cortx.data.serviceDomain" $)) -}}
 {{- end -}}
-{{- $dataHaxPort := 22001 -}}
-{{- $dataIosPort := 21001 -}}
-{{- $dataConfdPort := 22002 -}}
+{{- $serverHostnames := list -}}
+{{- if .Values.cortxserver.enabled -}}
+{{- range $i := until (int .Values.cortxserver.replicas) -}}
+{{- $serverHostnames = append $serverHostnames (printf "%s-%d.%s" (include "cortx.server.fullname" $) $i (include "cortx.server.serviceDomain" $)) -}}
+{{- end -}}
+{{- end -}}
+{{- $haxPort := 22001 -}}
```

Comment:
Can these port numbers be pulled from anywhere (values.yaml) so that they can be uniformly applied and changed only in one place?

Reply:
I was opting not to, since I don't see the point of making it configurable. We could certainly allow it though, if that's preferred.

Or I could make them template functions, which is more verbose, but puts them in one place.

Reply:
I'll go with the templates for now, we can always expose them in values.yaml as needed.